CMakeLists.txt

```diff
@@ -1,4 +1,8 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.0)
```

Comment:
People build this on a lot of platforms, some funky and old like S390, ancient Debian stable, embedded platform with old rootfs basis who have no interest in making the latest and greatest available.  From that point of view "it's better" the minimum CMake version just reflects what it needs.... the warning is preferable.  Documenting it might be better.